### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found few misspellings.

@niden Including code.

### CODE REVIEW

File: _posts/2021-12-25-merry-christmas-and-phalcon-5-0-0beta1-released.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ If we do have time, we will upgrade the sample applications (invo, vokuro etc.)
 - Changes to `Phalcon\Autoload\Loader`:
   - Renamed `registerClasses()` to `setClasses()`
   - Renamed `registerDirectories()` to `setDirectories()`
-  - Renamed `registertExtensions()` to `setExtensions()`
-  - Renamed `registertFiles()` to `setFiles()`
-  - Renamed `registertNamespaces()` to `setNamespaces()` [#15797](https://github.com/phalcon/cphalcon/issues/15797)
+  - Renamed `registeredExtensions()` to `setExtensions()`
+  - Renamed `registeredFiles()` to `setFiles()`
+  - Renamed `registeredNamespaces()` to `setNamespaces()` [#15797](https://github.com/phalcon/cphalcon/issues/15797)
```
Comment (szepeviktor) [inline _posts/2021-12-25-merry-christmas-and-phalcon-5-0-0beta1-released.md:59]: Not sure here.

File: _posts/2021-12-25-merry-christmas-and-phalcon-5-0-0beta1-released.md [hunk 2]
```diff
@@ -54,9 +54,9 @@ If we do have time, we will upgrade the sample applications (invo, vokuro etc.)
 - Changes to `Phalcon\Autoload\Loader`:
   - Renamed `registerClasses()` to `setClasses()`
   - Renamed `registerDirectories()` to `setDirectories()`
-  - Renamed `registertExtensions()` to `setExtensions()`
-  - Renamed `registertFiles()` to `setFiles()`
-  - Renamed `registertNamespaces()` to `setNamespaces()` [#15797](https://github.com/phalcon/cphalcon/issues/15797)
+  - Renamed `registerExtensions()` to `setExtensions()`
+  - Renamed `registerFiles()` to `setFiles()`
+  - Renamed `registerNamespaces()` to `setNamespaces()` [#15797](https://github.com/phalcon/cphalcon/issues/15797)
```
Comment (szepeviktor) [inline _posts/2021-12-25-merry-christmas-and-phalcon-5-0-0beta1-released.md:59]: https://github.com/phalcon/cphalcon/blob/03653b437245052d8300224d15aa02f802d373a5/CHANGELOG-5.0.md?plain=1#L483-L487

**Review by Arhell — APPROVED** [review]

**Comment by niden** [comment]
Thank you @Arhell

**Comment by Arhell** [comment]
Thanks to the author

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**